src/autofit/ft-hb.c

```diff
@@ -61,7 +61,13 @@ void ft_hb_funcs_init ( struct  AF_ModuleRec_ *af_module )
      * holds onto global variables, and closing the library
      * will cause them to be invalidated.
      */
-    lib = dlopen( "libharfbuzz.so.0", RTLD_LAZY | RTLD_GLOBAL );
+#ifdef __APPLE__
+#define LIBHARFBUZZ "libharfbuzz.dylib"
+#else
+#define LIBHARFBUZZ "libharfbuzz.so"
+#endif
```

Comment:
Needs Windows support as well. Not sure how things work on Windows, e.g. would putting HarfBuzz DLL next to the FreeType DLL simply work or not.

Reply:
Just pushed untested Windows support. I haven't tried to cross-compile it yet.

Reply:
> Needs Windows support as well. Not sure how things work on Windows, e.g. would putting HarfBuzz DLL next to the FreeType DLL simply work or not.

My experience, at least on Wine, is that the DLL should either be in system dirs, or next to the *binary*. FreeType DLL is probably in one of those locations, so HarfBuzz DLL next to it should work.

Reply:
Calling `dlclose` on Linux was causing crashes. I convinced myself that it's a bad idea to `dlclose` since `RTLD_GLOBAL`. Not sure on Windows. Last reports we had `FreeLibrary` on Windows was *also* crashing. So, I'll leave it as is.